Formula/libtiff.rb

```diff
@@ -34,12 +36,14 @@ def install
     ENV.cxx11 if build.cxx11?
 
     jpeg = Formula["jpeg"].opt_prefix
+    lzma = Formula["xz"].opt_prefix
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--without-x",
-                          "--disable-lzma",
                           "--with-jpeg-include-dir=#{jpeg}/include",
-                          "--with-jpeg-lib-dir=#{jpeg}/lib"
+                          "--with-jpeg-lib-dir=#{jpeg}/lib",
+                          "--with-lzma-include-dir=#{lzma}/include",
```

Comment:
Given you've made this optional, you shouldn't automatically throw it into the configure arguments as it'll break.